homeassistant/components/climate/daikin.py

```diff
@@ -76,7 +85,7 @@ def __init__(self, api):
         self._force_refresh = False
         self._list = {
             ATTR_OPERATION_MODE: list(
-                map(str.title, set(HA_STATE_TO_DAIKIN.values()))
+                map(str, set(DAIKIN_TO_HA_STATE.values()))
```

Comment:
Why do we need the `str` copy?

Reply:
Making it a set doesn't seem necessary either. We know what the values are and they are unique. The dictionary is a constant so it won't change unless we change it.

Reply:
For this part I made just little modification to make it work properly, but the logics is the one from the original commit for the daikin component.

Reply:
Since we're here, please clean it up.

Reply:
Change this to:
```py
self._list = {
    ATTR_OPERATION_MODE: list(HA_STATE_TO_DAIKIN),
    ...
}
```